<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitDialog</class>
 <widget class="QDialog" name="InitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预分配</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>首地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="endLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>末地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="endEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>180</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
